numpy/core/tests/test_umath.py

```diff
@@ -1861,8 +1862,8 @@ class A(object):
             def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
                 return self, ufunc, method, inputs, kwargs
 
+        inner1d = ncu_tests.inner1d
```

Comment:
For some reason, that did not work with just the `python numpy/core/tests/test_umath.py` -- but possibly my installation was half-screwed or so.